coalib/bearlib/aspects/Redundancy.py

```diff
@@ -0,0 +1,146 @@
+from coalib.bearlib.aspects import Aspect, AspectDocumentation
+
+
+class Redundancy(Aspect):
+    docs = AspectDocumentation(
+        definition="This aspect describes redundancy in your source code.",
+        cause="Redundancies are often caused by forgetting to remove "
+              "something during refactorings.",
+        example="if False:\n    do_something()",
```

Comment:
I think we should make up a better example here, put this in more or less as a placeholder